CODEOWNERS

```diff
@@ -0,0 +1,82 @@
+# Catch-all
+* @tgraf @manalibhutiyani
+
+# BPF Data Path
+bpf/* @joestringer @borkmann @jrfastab @manalibhutiyani
```

Comment:
I will create github groups to make this easier. What convention do we want to use for naming?

@grp-bpf? @bpf? @team-bpf?

Reply:
My vote: @team-bpf

Reply:
+1, for prefixing with "team-"